src/components/grid/grid.ts

```diff
@@ -617,6 +617,7 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
   }
 
   async paste(ev: ClipboardEvent) {
+    ev.preventDefault();
```

Comment:
this prevents from pasting in all regular inputs :/

Reply:
Indeed put in a very pad place :/